imports/plugins/included/product-detail-simple/client/components/productField.js

```diff
@@ -130,16 +130,22 @@ class ProductField extends Component {
       return this.renderTextField();
     }
 
+    const classNames = classnames({
+      pdp: true,
+      field: true,
+      [this.fieldName]: this.fieldName
```

Comment:
This should be a Boolean `true` or `false` however you decide to calculate that, or literal `true` or `false`.

`[this.fieldName]:  true|false|typeof this.fieldName === "string"|whatever`

Reply:
you mean something like this?

```
[this.fieldName]: !!this.fieldName
```

Reply:
@mikemurray  I did this, because even
```
classnames({a: true, undefined})
```
yields the correct class names. And I think we can't guarantee that this.fieldName is given (that comes from template definition, right)?!